packaging/linux/common/khiops_env/khiops_env.in

```diff
@@ -64,9 +65,18 @@ get_script_dir() {
 @IS_CONDA_VAR@
 @SET_KHIOPS_DRIVERS_PATH@
 
+if [ -z "$KHIOPS_MPI_VERBOSE" ]; then
+    KHIOPS_MPI_VERBOSE="false"
```

Comment:
Shouldn't it be the other way round: `KHIOPS_MPI_VERBOSE=true` => there is _no_ `--quiet`; else, `--quiet` is added?

Reply:
yes indeed... sorry